test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Error.Tests.ps1

```diff
@@ -66,6 +69,9 @@ Describe "Write-Error Tests" -Tags "CI" {
         #InvocationInfo verification
         $e.InvocationInfo | Should -Not -BeNullOrEmpty
         $e.InvocationInfo.MyCommand.Name | Should -BeNullOrEmpty
+
+        #IsTerminating verification
+        $e.IsTerminating | Should -BeFalse
```

Comment:
Seems like there should be more explicit tests here.  Whether an Error is terminating or non-terminating depends on the state of that Error.  If it is handled, then it is considered non-terminating.  A non-terminating ErrorRecord can be terminating by changing ErrorAction, but then can be non-terminating again by using Trap with continue.